src/components/Editor/DrawingBoard/Canvas/Board.ts

```diff
@@ -1,19 +1,24 @@
+import { Client } from 'yorkie-js-sdk';
 import { ToolType, Color } from 'features/boardSlices';
 import { Point, Shape } from 'features/docSlices';
+import { Peer } from 'features/peerSlices';
 import EventDispatcher from 'utils/eventDispatcher';
 
 import CanvasWrapper from './CanvasWrapper';
 import { drawLine } from './line';
 import { drawRect } from './rect';
 import { addEvent, removeEvent, touchy, TouchyEvent } from './dom';
 import { Worker, LineWorker, EraserWorker, RectWorker, SelectorWorker } from './Worker';
+import NoneWorker from './Worker/NoneWorker';
 
 enum DragStatus {
   Drag,
   Stop,
 }
 
-export default class Container extends EventDispatcher {
+export default class Board extends EventDispatcher {
```

Comment:
It looks like the Container has changed. Please update the design documentation as well.